backend/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 
+spring.h2.console.enabled=true
```

Comment:
Nem mindenki tudja a default user/pass-t, ne kelljen minden alkalommal utána nézni, inkább adjunk sajátot (személyes véleményem csak)

Reply:
A defaultot be is írja alapból. Csak rá kell nyomni az ok -ra a belépéshez.